cmd/gctcli/commands.go

```diff
@@ -2269,6 +2302,95 @@ func cancelAllOrders(c *cli.Context) error {
 	return nil
 }
 
+func modifyOrder(c *cli.Context) error {
+	if c.NArg() == 0 && c.NumFlags() == 0 {
+		return cli.ShowCommandHelp(c, "modifyorder")
+	}
+
+	// Parse positional arguments.
+	var exchangeName string
+	var orderID string
+	var currencyPair string
+	var assetType string
+
+	if c.IsSet("exchange") {
+		exchangeName = c.String("exchange")
+	} else {
+		exchangeName = c.Args().First()
+	}
+	if !validExchange(exchangeName) {
+		return errInvalidExchange
+	}
+
+	if c.IsSet("asset") {
+		assetType = c.String("asset")
+	} else {
+		assetType = c.Args().Get(1)
+	}
+	assetType = strings.ToLower(assetType)
+	if !validAsset(assetType) {
+		return errInvalidAsset
+	}
+
+	if c.IsSet("pair") {
+		currencyPair = c.String("pair")
+	} else {
+		currencyPair = c.Args().Get(2)
+	}
+	if !validPair(currencyPair) {
+		return errInvalidPair
+	}
+
+	p, err := currency.NewPairDelimiter(currencyPair, pairDelimiter)
+	if err != nil {
+		return err
+	}
+
+	if c.IsSet("order_id") {
+		orderID = c.String("order_id")
+	} else {
+		orderID = c.Args().Get(3)
+	}
+
+	// Parse optional flags.
+	var price float64
+	var amount float64
+
+	if c.IsSet("price") {
+		price = c.Float64("price")
+	}
+	if c.IsSet("amount") {
+		amount = c.Float64("amount")
+	}
```

Comment:
I agree with Shazbert. Please do a check where if price and amount is 0, then don't proceed

Reply:
What if the client wants to modify just one of them? This is already supported in OrderManager.Modify.

Reply:
I mean an 
`If amount == 0 && price == 0 { return err } ` 
This is just a simple client side check to ensure that the request is valid and saving on a call to the application when invalid